models/patient.js

```diff
@@ -0,0 +1,131 @@
+const { Model } = require('sequelize');
```

Comment:
@komerela I know we made a bunch of edits to this model yesterday, so I'll skip reviewing it.